.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: Python package
+
+on: [push, pull_request]
+
+jobs:
+
+  build:
+    name: Build for (${{ matrix.python-version }}, ${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
When using `matrix`, I usually first use a "smoke test" on a specific platform and python verison. If the smoke test succeed, then do the test on the matrix. With that, it will save time and computation resource, otherwise you have to wait the tests on all matrix options fail when there is a bug in the code. Check this example (line 6-25 and 29) see how to use smoke test: https://github.com/DeepRank/pdb2sql/blob/b46ba1d3aa7abd496309c05e1fc5fcdd881387fb/.github/workflows/build_test.yml

Reply:
I completely removed matrix testing, see https://github.com/PerfectFit-project/virtual-coach-server/pull/72 for rationale.